examples/dynamic-bridge-app/linux/include/data-model/DataModel.h

```diff
@@ -23,40 +40,34 @@ namespace DataModel {
 // iterator_type must conform to LegacyForwardIterator
 // The contained type must be default-constructible
 template <typename X>
-class IsList
+class AsList : public X
```

Comment:
Is there a way to validate some of the above expectations using static_assert here?

Reply:
It's pretty ugly but yes